acl_setup.php

```diff
@@ -102,6 +107,8 @@
      // xl('Pharmacy Dispensary')
  $gacl->add_object('admin', 'ACL Administration'              , 'acl'      , 10, 0, 'ACO');
      // xl('ACL Administration')
+ $gacl->add_object('admin', 'Multipledb'                      ,'multipledb', 10, 0, 'ACO');
```

Comment:
A multiple DB is a way to connect to multiple databases . This module after configured in the global security  param  and inside the module hook  can let you connect  to other DB.